cmiles/_cmiles_oe.py

```diff
@@ -33,7 +33,10 @@ def mol_from_json(symbols, connectivity, geometry, permute_xyz=False):
     for bond in connectivity:
         a1 = molecule.GetAtom(oechem.OEHasAtomIdx(bond[0]))
         a2 = molecule.GetAtom(oechem.OEHasAtomIdx(bond[1]))
-        molecule.NewBond(a1, a2, bond[-1])
+        bond_order = bond[-1]
+        if not isinstance(bond_order, int) and not bond_order.is_integer():
```

Comment:
Good thing to check. I had just run into un-`kekulize`d molecule problems last week.